contracts/BaseLaunchpeg.sol

```diff
@@ -62,18 +62,31 @@ abstract contract BaseLaunchpeg is
     /// @notice Token URI before the collection reveal
     string public override unrevealedURI;
 
-    /// @notice The amount of NFTs each allowed address can mint during the allowlist mint
+    /// @notice The amount of NFTs each allowed address can mint during
+    /// the pre-mint or allowlist mint
     mapping(address => uint256) public override allowlist;
 
+    // @notice The no. of NFTs pre-minted by the user address
+    mapping(address => uint256) public override numberPreMinted;
```

Comment:
can we say `userAddressToAmountPreMinted` or something like this ? it makes it clearer it's a mapping + we usually say "amount" and not "number" when we want to represent a quantity of NFTs

Reply:
updated!